src/RSCPL.xml

```diff
@@ -0,0 +1,473 @@
+        </li>
+        <li>
+          <b>(b)</b>
+          <p>any software, hardware, or device provided to You by the Participant, other than such
+             Participant's Original Code or Contributor Version, directly or indirectly infringes any
+             patent, then any rights granted to You by such Participant under Sections 2.1(b) and 2.2(b)
+             are revoked effective as of the date You first made, used, sold, distributed, or had made,
+             Original Code or the Modifications made by that Participant.</p>
+        </li>
+        </list>
+        </li>
+        <li>
+          <b>8.3.</b>
+          <p>If You assert a patent infringement claim against Participant alleging that such Participant's
+             Original Code or Contributor Version directly or indirectly infringes any patent where such
+             claim is resolved (such as by license or settlement) prior to the initiation of patent
+             infringement litigation, then the reasonable value of the licenses granted by such Participant
+             under Sections 2.1 or 2.2 shall be taken into account in determining the amount or value of
+             any payment or license.</p>
+        </li>
+        <li>
+          <b>8.4.</b>
+          <p>In the event of termination under Sections 8.1 or 8.2 above, all end user license agreements
+             (excluding distributors and resellers) which have been validly granted by You or any
+             distributor hereunder prior to termination shall survive termination.</p>
+        </li>
+        </list>
+        </li>
+        <li>
+          <b>9.</b>
+          <p>Limitation of Liability. 
+            <br/>UNDER NO CIRCUMSTANCES AND UNDER NO LEGAL THEORY, WHETHER TORT (INCLUDING NEGLIGENCE),
+                 CONTRACT, OR OTHERWISE, SHALL RSV, ANY CONTRIBUTOR, OR ANY DISTRIBUTOR OF GOVERNED
+                 CODE, OR ANY SUPPLIER OF ANY OF SUCH PARTIES, BE LIABLE TO YOU OR ANY OTHER PERSON FOR
+                 ANY DIRECT, INDIRECT, SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES OF ANY CHARACTER
+                 INCLUDING, WITHOUT LIMITATION, DAMAGES FOR LOSS OF GOODWILL, WORK STOPPAGE, COMPUTER
+                 FAILURE OR MALFUNCTION, OR ANY AND ALL OTHER COMMERCIAL DAMAGES OR LOSSES, EVEN IF
+                 SUCH PARTY SHALL HAVE BEEN INFORMED OF THE POSSIBILITY OF SUCH DAMAGES. THIS
+                 LIMITATION OF LIABILITY SHALL NOT APPLY TO LIABILITY FOR DEATH OR PERSONAL INJURY
+                 RESULTING FROM SUCH PARTY'S NEGLIGENCE TO THE EXTENT APPLICABLE LAW PROHIBITS SUCH
+                 LIMITATION. SOME JURISDICTIONS DO NOT ALLOW THE EXCLUSION OR LIMITATION OF INCIDENTAL
+                 OR CONSEQUENTIAL DAMAGES, SO THAT EXCLUSION AND LIMITATION MAY NOT APPLY TO YOU. TO
+                 THE EXTENT THAT ANY EXCLUSION OF DAMAGES ABOVE IS NOT VALID, YOU AGREE THAT IN NO
+                 EVENT WILL RSVS LIABILITY UNDER OR RELATED TO THIS AGREEMENT EXCEED FIVE THOUSAND
```

Comment:
It looks like “RSVS” should be “RSV'S”.  I see no other occurrences of “RSVS” in this license, and this case here certainly looks possessive.  And [one version of the historical upstream source][1] is using *something* there, although they don't seem to have declared the right encoding.

[1]: http://wayback.archive.org/web/20060715140826/http://www.risource.org/RPL/RPL-1.0A.shtml